docker/Dockerfile_aarch64_ascend

```diff
@@ -115,57 +103,25 @@ SHELL ["/bin/bash", "-c"]
 RUN echo "source /usr/local/Ascend/ascend-toolkit/set_env.sh" >> ~/.bashrc && \
     . ~/.bashrc
 
-WORKDIR /deeplink
-RUN echo -e "diff --git a/impl/ascend_npu/CMakeLists.txt b/impl/ascend_npu/CMakeLists.txt\n\
-index e684c59..f1cd8d4 100755\n\
---- a/impl/ascend_npu/CMakeLists.txt\n\
-+++ b/impl/ascend_npu/CMakeLists.txt\n\
-@@ -14,6 +14,11 @@ FetchContent_Declare(op_plugin\n\
- FetchContent_MakeAvailable(op_plugin)\n\
- message(STATUS \"op-plugin download done\")\n\
- \n\
-+add_custom_target(patch_op_plugin_code\n\
-+  COMMAND sed -i 's/GetOpApiLibHandler\(GetCustOpApiLibName\(\)\)/nullptr/' \${op_plugin_SOURCE_DIR}/op_plugin/utils/op_api_common.h\n\
-+  BYPRODUCTS \${op_plugin_SOURCE_DIR}/op_plugin/utils/op_api_common.h\n\
-+)\n\
-+\n\
- add_custom_target(op_plugin_gen\n\
-   COMMAND cd \${op_plugin_SOURCE_DIR} && bash ./gencode.sh 2.1 python\n\
-   BYPRODUCTS \${op_plugin_SOURCE_DIR}/op_plugin/OpInterface.h \${op_plugin_SOURCE_DIR}/op_plugin/OpInterface.cpp\n\
-@@ -253,7 +258,7 @@ endif()\n\
- set(THIRD_PARTY_INCLUDE_DIRS \${CMAKE_CURRENT_SOURCE_DIR}/../third_party/half/include)\n\
- \n\
- add_library(\${DEVICEIMPL} SHARED \${IMPL_SRC})\n\
--add_dependencies(\${DEVICEIMPL} op_plugin_gen)\n\
-+add_dependencies(\${DEVICEIMPL} op_plugin_gen patch_op_plugin_code)\n\
- set_target_properties(\${DEVICEIMPL} PROPERTIES SUFFIX \".so\")\n\
- target_include_directories(\${DEVICEIMPL} PRIVATE \${ASCEND_DIR}/ascend-toolkit/latest/include/aclnn)\n\
- target_include_directories(\${DEVICEIMPL} SYSTEM PUBLIC \${THIRD_PARTY_INCLUDE_DIRS})\n" > /deeplink/warning.patch
-
-# deeplink
-RUN git clone https://github.com/DeepLink-org/deeplink.framework.git && \
-    cd deeplink.framework/dipu && \
-    git checkout ${DEEPLINK_TAG_OR_COMMIT} && \
-    git submodule update --init --recursive && \
-    git -C ./third_party/DIOPI apply /deeplink/warning.patch && \
-    DIPU_DEVICE=ascend python setup.py develop && \
-    rm -rf /root/.cache/pip
-
-# deeplink_ext
-RUN git clone https://github.com/DeepLink-org/DeepLinkExt.git && \
-    cd DeepLinkExt && \
-    git checkout ${DEEPLINKEXT_TAG_OR_COMMIT} && \
-    DIPU_REPO=/deeplink/deeplink.framework/dipu DIPU_ROOT=${DIPU_REPO}/torch_dipu \
-    DIOPI_PATH=${DIPU_REPO}/third_party/DIOPI/proto \
-    VENDOR_INCLUDE_DIRS=/usr/local/Ascend/ascend-toolkit/latest/include \
-    pip install -vv --no-build-isolation -e . && \
-    rm -rf /root/.cache/pip
+# dlinfer
+# transformers>=4.41.0 is required for internlm2 model
+# timm is required for internvl2 model
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip3 install transformers>=4.41.0 timm && \
+    pip3 install dlinfer-ascend==0.1.0
 
 # lmdeploy
-WORKDIR /workspace
-RUN git clone https://github.com/InternLM/lmdeploy.git && \
-    cd lmdeploy && \
-    git checkout ${LMDEPLOY_TAG_OR_COMMIT} && \
+WORKDIR /opt/lmdeploy
+
+COPY autotest /opt/lmdeploy/autotest
```

Comment:
where autotest comes from?

Reply:
<img width="1177" alt="image" src="https://github.com/user-attachments/assets/97db3488-20e0-4c61-b359-8be987142787">
Red rectangle selected region in picture, just a dir under repo root dir.